diff --git a/src/base_realsense_node.cpp b/src/base_realsense_node.cpp
index 5e4e6e7..59cf468 100755
--- a/src/base_realsense_node.cpp
+++ b/src/base_realsense_node.cpp
@@ -784,7 +784,7 @@ void BaseRealSenseNode::SetBaseStream()
 
 void BaseRealSenseNode::publishPointCloud(rs2::points pc, const rclcpp::Time& t, const rs2::frameset& frameset)
 {
-    std::string frame_id = (_align_depth_filter->is_enabled() ? OPTICAL_FRAME_ID(COLOR) : OPTICAL_FRAME_ID(DEPTH));
+    std::string frame_id =  OPTICAL_FRAME_ID(DEPTH);
     _pc_filter->Publish(pc, t, frameset, frame_id);
 }
